tests/test_floss.py

```diff
@@ -193,3 +192,73 @@ def test_floss():
         npt.assert_almost_equal(left_P, right_P)
         npt.assert_almost_equal(left_I, right_I)
         npt.assert_almost_equal(left_T, right_T)
+
+
+test_data_nan = np.insert(
+    np.random.uniform(-1000, 1000, [64]), [i for i in range(30, 60, 5)], np.nan
+)
+test_data_inf = np.insert(
+    np.random.uniform(-1000, 1000, [64]), [i for i in range(30, 60, 5)], np.inf
+)
+
+
+@pytest.mark.parametrize("data", [(test_data_nan), (test_data_inf)])
```

Comment:
@JosephTLucas I apologize for the late curveball here so feel free to ignore and I can handle this after the merge if it's too much to ask. Usually, when we do `nan`/`inf` testing, we like to define the substitution of real float values at the top like:

https://github.com/TDAmeritrade/stumpy/blob/164368e97e2acae5f882a3acb5e7caefc144b7a0/tests/test_stump.py#L9-L21

and then the unit test itself looks like:

https://github.com/TDAmeritrade/stumpy/blob/164368e97e2acae5f882a3acb5e7caefc144b7a0/tests/test_stump.py#L176-L197

For consistency with other tests, would you mind changing this to:

```
# Add this at the top of `test_floss.py`
substitution_locations = [(slice(0, 0), 0, -1, slice(1, 3), [0, 3])] 
substitution_values = [np.nan, np.inf] 
```

and then in the test:

```
@pytest.mark.parametrize("substitute", substitution_values) 
@pytest.mark.parametrize("substitution_locations", substitution_locations) 
def test_floss_nan_inf(substitute, substitution_locations): 
    T = np.random.uniform(-1000, 1000, [64])
    m = 5
    data = T.copy()

    for substitution_location in substitution_locations:
        data[:] = T[:]
        data[substitution_location] = substitute
        old_data = data[:30]
        n = old_data.shape[0]
```

Reply:
We've found that testing like this (i.e., moving the `nan`/`inf` around to multiple substitution locations but keeping the same original data within a run) has covered a lot of edge cases and is also consistent across other tests. Again, feel free to let me know if you'd rather I handle it after the merge. Happy to do so!

Reply:
Of course, I'll make that change!

Reply:
Great! Thank you and please let me know if you have any questions.